### PR TITLE
React 0.13 / Babel

### PR DESCRIPTION
**Pull request by solomonhawk**
:point_up: 

:point_down:

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ let Ink = React.createClass({
   },
 
   pushBlot(timeStamp, clientX, clientY) {
-    let el = this.getDOMNode()
+    let el = React.findDOMNode(this)
```
Comment (nhunzaker) [inline src/index.jsx]: This will :fire: < 0.13

File: package.json [hunk 2]
```diff
@@ -28,19 +28,18 @@
...
+    "react-tools": "^0.13.0",
     "transform-loader": "^0.2.1",
-    "webpack": "^1.4.13",
-    "webpack-dev-server": "^1.6.6"
+    "webpack": "^1.7.2",
+    "webpack-dev-server": "^1.7.0"
   },
   "peerDependencies": {
-    "react": ">= 0.12.x"
+    "react": ">= 0.13.x"
```
Comment (nhunzaker) [inline package.json]: This should still be `0.12.x`. We don't specifically rely on any 0.13 feature

**Comment by nhunzaker** [comment]
Just dependency stuff, otherwise :+1:

**Comment by averyvery** [comment]
ez :+1:

File: package.json (hunk 2 above)
Comment (nhunzaker) [inline package.json]: @solomonhawk I will merge if you lower this back down to `0.12.0`

**Pull request merged**